framer/LayerDraggable.coffee

```diff
@@ -570,11 +573,13 @@ class exports.LayerDraggable extends BaseClass
 
 	_stopSimulation: =>
 		@_isAnimating = false
+		@_isMoving = false
+
 		return unless @_simulation
 		@_simulation?.x.stop()
 		@_simulation?.y.stop()
 		@_simulation = null
-		@emit(Events.Move, @layer.point)
+		#@emit(Events.Move, @layer.point)
```

Comment:
Did some spelunking, this `emit` was introduced here: 8ccf6c65fa5b0744c8b0a399a30481071d4f1df0. If I understand the intent correctly, I think this is to make sure that if you use an `onMove` event listener, this code guarantees that you always get the final state also. The change was made >18 months though, so things might have changed, but I haven’t tested or reviewed code to make sure that it’s not still necessary.